onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -294,7 +294,7 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       transformers.emplace_back(std::make_unique<ConvActivationFusion>(cpu_cuda_rocm_acl_armnn_eps));
 
       transformers.emplace_back(std::make_unique<GeluFusion>(cpu_cuda_dml_rocm_eps));
-      transformers.emplace_back(std::make_unique<LayerNormFusion>(cpu_cuda_dml_rocm_eps));
+      transformers.emplace_back(std::make_unique<LayerNormFusion>(cpu_cuda_rocm_eps));
```

Comment:
We can't just disable this fusion. This would significantly regress perf for important models that we track like LLMs and stable diffusion. It can't be that hard to fix the actual root cause until the next ORT release.

Reply:
You are on a mail regarding this.

Reply:
Reverted, the root cause was in dml, so an update will be required... No need to disable this test.
Removed.